generators/spa/templates/common/Controllers/BusinessController.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+Copyright Upendo Ventures, LLC 
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), 
+to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, 
+and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+DEALINGS IN THE SOFTWARE.
+*/
+
+using DotNetNuke.Entities.Modules;
+using DotNetNuke.Services.Tokens;
+using DotNetNuke.UI.Modules;
+using System.Collections.Generic;
+using System.Web.UI;
+
+namespace <%= fullNamespace %>.Controllers
+{
+    public class BusinessController : ICustomTokenProvider
+    {
+        public IDictionary<string, IPropertyAccess> GetTokens(Page page, ModuleInstanceContext moduleContext)
```

Comment:
@VAlmea What is this method used for?

Reply:
This method returns an object with all the information needed to update the headers of HTTP requests. The same approach is used in the page-manager project.